utils/pswdgen/generator.go

```diff
@@ -139,11 +139,17 @@ func Generate(config Config) (string, error) {
 
 func readTermChar() ([]byte, error) {
 	t, _ := term.Open("/dev/tty")
```

Comment:
Please add an error check for `term.Open("/dev/tty")`

Reply:
Fixed.